prover/witness_generator/src/basic_circuits.rs

```diff
@@ -680,7 +643,13 @@ async fn generate_witness(
 
         let vm_storage_oracle: VmStorageOracle<StorageView<PostgresStorage<'_>>, HistoryDisabled> =
             VmStorageOracle::new(storage_view.clone());
-        let storage_oracle = StorageOracle::new(vm_storage_oracle, storage_refunds);
+        let storage_oracle = StorageOracle::new(
+            vm_storage_oracle,
+            storage_refunds,
+            pubdata_costs.expect("pubdata costs should be present"),
+        );
+
+        let path = KZG_TRUSTED_SETUP_FILE.path().to_str().unwrap();
```

Comment:
nit: let's keep `.expect("Path to KZG trusted setup is not a UTF-8 string");` here